recipes/pypgx/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set name = "pypgx" %}
+{% set version = "0.4.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3907bed097fde74d16d011ae196f18fef5496d2a85274b143e772052ffee7aea
+
+build:
+  number: 0
+  entry_points:
+    - pypgx=pypgx.__main__:main
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python
+  run:
+    - fuc
+    - openjdk
+    - python
+    - scikit-learn
+
+test:
+  imports:
+    - pypgx
+    - pypgx.api
+    - pypgx.api.data
+    - pypgx.cli
+    - pypgx.sdk
+  commands:
+    - pypgx --help
+
+about:
+  home: "https://github.com/sbslee/pypgx"
+  license: MIT
+  license_family: MIT
```

Comment:
One small thing. Can you add `license_file` here please? Otherwise all looks good to me.

Reply:
Hi @simonbray, thank you for the review! I have a file named `LICENSE` in the `pypgx` repo, so I was going to add `license_file: LICENSE`. However, whenever I try to make edits to the file, it would not let me saying I "must be on a branch" to make changes. I'm not sure why I cannot suddenly make edits. Is this because I deleted the forked repo? I deleted it because I thought once I have a PR I don't need to keep the forked repo anymore. In any case, would it be possible for you to add the said line for me as a reviewer? Or is this PR doomed and do I need to start over?

Reply:
Yes, it was a mistake to delete the forked repo unfortunately. I just went ahead and merged, the missing entry does not matter that much. (You could create a new PR to add it if you really want to).

Reply:
@simonbray, thank you for the prompt response and being understanding. Definitely learned my lesson (come to think about it, it was foolish to delete a forked repo that has an active PR)! I will make the change during the first `autobump` when the next version is released.

Reply:
Cool, that sounds like a good idea